config/application.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "boot"
```

Comment:
Whats this file about?

Reply:
This file allows you to pull all gems you required in `Gemfile` into your project by using `Bundler`. `Bundler` looks into the Gemfile for gems belonging to specified groups and calls `require` on the gems.